raiden/ui/app.py

```diff
@@ -107,24 +102,6 @@ def fetch_available_matrix_servers(
     transport_config.available_servers = available_servers
 
 
-def setup_matrix(
```

Comment:
I'm not sure if it is the right thing to remove the function here. Did you implement this functionality somewhere else? Otherwise we would keep sending messages to either PFS or MS even if it is disabled. 
This leads me to the question, do we still have or rather support `RoutingMode.PRIVATE`?

Reply:
This is just the initial configuration setup based on the `routing-mode` and `monitoring-enabled`, causing the MatrixTransport to connect to the services broadcast rooms - nothing else was done in this function.

> This leads me to the question, do we still have or rather support RoutingMode.PRIVATE?

If the mode is private or monitoring is disabled, the corresponding config parameters will be checked throughout the codebase (`raiden.services.send_pfs_update` / `raiden.services.update_monitoring_service_from_balance_proof`) and the calls to `broadcast` will be omitted.

Reply:
great